program/convert-string-to-pascalcase/ConvertStringToPascalcase.kt

```diff
@@ -0,0 +1,23 @@
+import java.util.Scanner
+
+fun main() {
+    print("Enter String : ")
+    println()
+    var min = readLine()
+
+    println("Pasacl Case: " + ConvertStringToPascalcase(min!!.split(" ")))
+
+}
+
+private fun ConvertStringToPascalcase(args: List<String>): String {
+    val buffer = StringBuffer()
+    for((index,day) in args.withIndex()){
+        if(index == 0) {
+            buffer.append(day.get(0).uppercase()).append(day.subSequence(1,day.length))
+        } else {
+            buffer.append(" ").append(day.get(0).uppercase()).append(day.subSequence(1,day.length))
+        }
+    }
```

Comment:
Getting error while executing this program 

```
OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.
prog.kt:16:38: error: unresolved reference: uppercaseChar
            buffer.append(day.get(0).uppercaseChar()).append(day.subSequence(1,day.length))
                                     ^
prog.kt:18:50: error: unresolved reference: uppercaseChar
            buffer.append(" ").append(day.get(0).uppercaseChar()).append(day.subSe
```

Reply:
work fine on https://www.tutorialspoint.com/online_kotlin_compiler.php